docs/resources/libraries.md

```diff
@@ -0,0 +1,208 @@
+---
+slug: libraries
+title: Libraries
+tags: [Libraries]
+keywords: [Libraries, Share, Reuse, Collaborate, Modularize, Dependency]
+description: Learn how to share and reuse entire FlutterFlow projects suing libraries.
+sidebar_position: 4
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Libraries
+
+The Libraries feature enables you to share and reuse entire FlutterFlow projects as libraries across multiple projects. This allows teams and developers to modularize their apps by creating shared libraries that include components, API calls, custom code, and more. By using libraries, development becomes more efficient and scalable.
+
+Imagine you're building an e-commerce app, and different teams are working on various features. One team develops a complex payment system. By using the Libraries, they can publish the payment system as a reusable library and allow other teams to easily import and integrate it into multiple projects without duplicating development efforts.
+
+![libraries.avif](imgs/libraries.avif)
+
+### Why Libraries
+
+Previously, FlutterFlow offered several methods to share resources between projects, such as team code libraries, design systems, API libraries, and by leveraging marketplace items. However, these methods had limitations, including the inability to share custom data types or custom functions alongside components or API calls and the absence of version control.
+
+With the Libraries, you can publish the complete FlutterFlow project as a library and import it as a dependency into other projects.
+
+:::tip[possible usecases]
+
+- **Modular Development**: Build large-scale apps by separating them into smaller, independently managed projects (e.g., UI library, backend integrations, etc.).
+- **Team Collaboration**: Share reusable UI components, custom functions, or API integrations across multiple apps within a team.
+- **Community Sharing**: Publish libraries that can be imported and reused by the broader FlutterFlow community.
+
+:::
+
+### What’s Included When Importing a Library
+
+When you import a library into a FlutterFlow project, the following resources are accessible for use:
+
+- [Components](../resources/ui/components/intro-components.md)
+- [Data Types & Enums](../resources/data-representation/custom-data-types.md)
+- [App State Variables](../resources/data-representation/app-state.md)
+- [Constants](../resources/data-representation/constants.md)
+- [API Calls](../resources/control-flow/backend-logic/api/rest-api.md)
+- [Action Blocks](../resources/control-flow/functions/action-blocks.md)
+- [Custom Functions](../ff-concepts/adding-customization/custom-functions.md), [Actions](../resources/control-flow/functions/action-flow-editor.md), and [Widgets](../resources/ui/widgets/intro-widgets.md)
+- [Assets](../resources/projects/settings/general-settings.md#app-assets) (Note: These are not versioned)
+
+:::note
+
+Pages are still being worked on and may come in future updates.
```

Comment:
I think that proactively manages user expectations. For users who may want to create libraries with fully featured pages, it's helpful to let them know this feature is still in development. It can also reduce support requests or feature inquiries related to pages.